relayer/pkg/chainlink/ocr2/contract_transmitter.go

```diff
@@ -57,12 +56,12 @@ func (c *contractTransmitter) Transmit(
 	//    epoch_and_round
 	//    extra_hash
 	reportContext := utils.RawReportContext(reportCtx)
+	// NOTE: Ensure extra_hash is 31 bytes with first byte blanked out
+	// libocr generates a 32 byte extraHash but we need to fit it into a felt
+	reportContext[2][0] = 0
+
```

Comment:
nit: Let's please extract this in a single util `RawReportContext` method, a proxy to `evmutil.RawReportContext`

Reply:
Done